components/Order/adjustments.rst

```diff
@@ -89,31 +52,56 @@ Let's add a 5$ discount to the previous example.
 
     $order = new Order();
     $tshirt = new OrderItem();
-    $tshirt
-        ->setName('Awesome T-Shirt')
-        ->setUnitPrice(4999)
-    ;
+    $tshirt->setUnitPrice(4999);
 
     $shippingFees = new Adjustment();
     $shippingFees->setAmount(1000);
 
     $tax = new Adjustment();
-    $tax
-        ->setAmount(1150)
-        ->setLabel()
-        ->setNeutral(true)
-    ;
+    $tax->setAmount(1150);
+    $tax->setNeutral(true);
+
+    $discount = new Adjustment();
+    $discount->setAmount(-500);
+
+    $order->addItem($tshirt);
+    $order->addAdjustment($shippingFees);
+    $order->addAdjustment($tax);
+    $order->addAdjustment($discount);
+    $order->calculateTotal();
+    $order->getTotal();
+
+    // Output will be 5499.
+
+Locked Adjustments
+------------------
+
+You can also lock adjustments it will ensure that your adjustment won't be deleted from order or order item.
```

Comment:
Maybe there should be a comma before "it"? Or consider something like this:
You can also lock adjustments, this will ensure that adjustment won't be deleted from order or order item.